CoreScriptsRoot/Modules/Server/ClientChat/DefaultClientChatModules/MessageCreatorModules/ChannelEchoMessage.lua

```diff
@@ -0,0 +1,46 @@
+local MESSAGE_TYPE = "ChannelEchoMessage"
```

Comment:
Any chance you can write out an API for these modules? (What different hooks are there for the different parts?) 
(Once this is all set in stone, that is).